sys/log/full/src/log_fcb.c

```diff
@@ -190,47 +190,25 @@ log_fcb_start_append(struct log *log, int len, struct fcb_entry *loc)
     return (rc);
 }
 
-static int
-log_fcb_append(struct log *log, void *buf, int len)
-{
-    struct fcb *fcb;
-    struct fcb_entry loc;
-    struct fcb_log *fcb_log;
-    int rc;
-
-    fcb_log = (struct fcb_log *)log->l_arg;
-    fcb = &fcb_log->fl_fcb;
-
-    rc = log_fcb_start_append(log, len, &loc);
-    if (rc) {
-        goto err;
-    }
-
-    rc = flash_area_write(loc.fe_area, loc.fe_data_off, buf, len);
-    if (rc) {
-        goto err;
-    }
-
-    rc = fcb_append_finish(fcb, &loc);
-
-err:
-    return (rc);
-}
-
 /**
  * Calculates the number of message body bytes that should be included after
  * the entry header in the first write.  Inclusion of body bytes is necessary
  * to satisfy the flash hardware's write alignment restrictions.
  */
 static int
-log_fcb_hdr_body_bytes(uint8_t align)
+log_fcb_hdr_body_bytes(uint8_t align, uint8_t ext_len)
 {
     uint8_t mod;
+    uint32_t size;
+
+    size = ext_len;
 
     /* Assume power-of-two alignment for faster modulo calculation. */
     assert((align & (align - 1)) == 0);
+  
+    size += LOG_BASE_ENTRY_HDR_SIZE;
```

Comment:
Sort of a nitpick, but it seems extra complicated to assign `size` in two statements.  Why not just `size = LOG_BASE_ENTRY_HDR_SIZE + ext_len`?